src/libertem/udf/base.py

```diff
@@ -1169,6 +1394,26 @@ def get_backends(self) -> BackendSpec:
         '''
         return ('numpy', )
 
+    def forbuf(self, arr, target):
```

Comment:
This is a comment on the previous method `get_backends`, namely the docstring needs updating to reflect the fact the possible return values have changed, and to reflect the support for sparse + gpu.

Reply:
Thx, nice catch! Should be fixed now.